File: Problem-2/problem2.php
<?php

/*****************
*
* Sum the even-valued fibonacci numbers 
* less than or equal to 4 million
*
*****************/

function evenFibonacci($max)
{
    $evenFibs = array();
    $lastFib = 1;
    for ($i = 1; $i <= $max; $i += 0)
    {
        if ($i % 2 == 0)
            $evenFibs[] = $i;
        $temp = $i;
        $i = $temp + $lastFib;
        $lastFib = $temp;
    }
    return $evenFibs;
}

print array_sum(evenFibonacci(4000000));

File: Problem-2/problem2.py
def evenFibonacciSum(max):
    sum = 0
    lastFib = 1
    i = 1
    while i < max:
        if (i % 2 == 0):
            sum += i
        temp = lastFib
        lastFib = i
        i += temp
    print "The sum of all even fibonacci numbers less than %d is %d" % (max, sum)

evenFibonacciSum(4000000)